### PR TITLE
fix #55824 baseball.yahoo.co.jp

### LINKED ISSUE #55824: baseball.yahoo.co.jp
### Issue URL (Annoyance)
 [https://baseball.yahoo.co.jp/npb/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbaseball.yahoo.co.jp%2Fnpb%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | 3.5 nightly 1 (5)
Browser: | Chrome (data compression)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/55824
`###RSAB` is for this:
<details><summary>Screenshot:</summary>

baseball
</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -32,6 +32,8 @@ reddit.com#$#body.scroll-disabled { overflow: visible!important; position: stati
 !+ NOT_PLATFORM(ios, ext_android_cb)
 reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPage"] > div[class="NavFrame"] > div[class="XPromoPopup active"]
 !
+baseball.yahoo.co.jp,sports.yahoo.co.jp###RSAB
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/mobile-app.txt:35]: Please add soccer.yahoo.co.jp to this rule too.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 2]
```diff
@@ -32,6 +32,8 @@ reddit.com#$#body.scroll-disabled { overflow: visible!important; position: stati
...
+baseball.yahoo.co.jp,sports.yahoo.co.jp##.sn-modInterstitial
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/mobile-app.txt:36]: I think it's better to move this rule to popups.txt

Comment (Yuki2718) [reply AnnoyancesFilter/sections/mobile-app.txt]: Done both.

**Comment by Yuki2718** [comment]
Found the same popup on `soccer.yahoo.co.jp` but only on sub-pages of it.

**Comment by zloyden** [comment]
Relsoved conflicts, but both banners are the same. I will fix it manually.

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
Thanks!

**Pull request merged**